corehq/toggles/__init__.py

```diff
@@ -537,7 +537,7 @@ def all_toggles_by_name_in_scope(scope_dict, toggle_class=StaticToggle):
     result = {}
     for toggle_name, toggle in scope_dict.items():
         if not toggle_name.startswith('__'):
-            if isinstance(toggle, toggle_class):
+            if type(toggle) == toggle_class:
```

Comment:
nit: add comment to indicate why `isinstance` can't be used

Reply:
Added